src/Pelagos/Util/DatasetIndex.php

```diff
@@ -92,21 +92,26 @@ protected function buildQuery(array $termsFilters = array(), $text = null, $geoF
             if (preg_match_all($doiRegEx, $text, $matches)) {
                 $text = trim(preg_replace($doiRegEx, '', $text));
                 foreach ($matches[1] as $doi) {
+                    $doiBoolQuery = new Query\BoolQuery();
                     // Query against dataset DOIs.
                     $doiQuery = new Query\Nested();
                     $doiQuery->setPath('doi');
                     $doiQuery->setQuery(
                         new Query\MatchPhrase('doi.doi', $doi)
                     );
 
+                    $doiBoolQuery->addMust($doiQuery);
+                    $doiBoolQuery->setBoost(3.0);
```

Comment:
Why does the boost work here? Have tried it on the config file, but didn't work.

Reply:
safer here than the config as that's global anyway imho

Reply:
Try searhing for 10.7266/n7-fjhy-6w15 and you'll see it sorts 3rd instead of first. Are you sure the boost is working?

Reply:
I had a bad local testing environment. I can see it's working now.

Reply:
Index time boosting is deprecated as of 5.0.0, probably it is removed in the current version which we are using